mail_show_follower/views/res_config_settings.xml

```diff
@@ -10,12 +10,88 @@
             <div id="emails" position="inside">
                 <div class="col-12 col-lg-6 o_setting_box">
                     <div class="o_setting_left_pane">
-                        <field name="show_internal_users_cc" />
                     </div>
                     <div class="o_setting_right_pane">
-                        <label for="show_internal_users_cc" />
-                        <div class="text-muted" id="show_internal_users_cc">
-                            Add internal users in cc mails details
+                        <div class="content-group">
+                            <label
+                                for="show_internal_users_cc"
```

Comment:
There's something weird on how this is rendered. 🤔  It doesn't follow the pattern.

https://user-images.githubusercontent.com/973709/159226040-2c68ca07-c099-4ba7-838d-74e2ebe67c07.mp4

As you can see, the main checkbox is positioned at the left of the vertical bar. Then, the extra options appear or disappear below. Maybe you can improve this part a little drawing inspiration from other modules that do similar stuff.

Reply:
Done moving chekcbox to the right